### PR TITLE
[WEB-1855] - About View Redesign

### PR DESCRIPTION
**Pull request by RavenHursT**
https://goodparty.atlassian.net/browse/WEB-1855

### CODE REVIEW

File: app/(company)/about/components/BuildWithUsSection.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+const BuildWithUsHeaderAndBlurb = () => <div className="xl:col-span-2">
+    <h2 className="text-4xl mb-4 md:text-5xl leading-tight font-medium md:text-6xl leading-snug">Build
+    a better <br className="hidden md:inline" />democracy <br
+      className="md:hidden" /><span
+      className="text-tertiary-main">with us</span></h2>
+  <p className="text-xl font-medium leading-7 mb-8 md:max-w-[466px] md:text-2xl leading-8">Ready
+    to join the movement? Support candidates, run for office or join our Discord
+    community of like-minded individuals.</p>
+  </div>
```
Comment (tomer-tgp) [inline app/(company)/about/components/BuildWithUsSection.js:11]: are you using the same code linter as we do? is you IDE respects prettier?

File: app/(company)/about/components/BuildWithUsSection.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  <div className="max-w-screen-xl mx-auto">
+    <div className="hidden xl:grid grid-cols-4 gap-4">
+      <BuildWithUsHeaderAndBlurb />
+      <BuildWithUsGridItems />
+    </div>
+    <div className="xl:hidden">
+      <BuildWithUsHeaderAndBlurb />
+      <div className="md:grid md:grid-cols-3 md:gap-4">
+        <BuildWithUsGridItems />
+      </div>
```
Comment (tomer-tgp) [inline app/(company)/about/components/BuildWithUsSection.js:32]: I think you can use just a grid here if you give grid values to smaller screens

File: app/(company)/about/components/HopeSection.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+const HopeSection = () => <section className="relative
+  bg-mint-200
+  px-4
+  py-8
+  md:px-24
+  md:py-24
+  xl:px-0
+  xl:mx-auto">
+  <div className="max-w-screen-xl mx-auto">
```
Comment (tomer-tgp) [inline app/(company)/about/components/HopeSection.js:53]:
```
<MaxWidth>
```

File: app/(company)/about/components/HopeSection.js [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+      building the independent movement with local
+      elections in 2024</p>
+    <div className="md:grid md:grid-cols-3 md:gap-x-4 md:gap-y-16">
+      {
+        HOPE_STATS.map(
+          ({ percentage, blurb }, key) => <HopeStat key={key}
+            percentage={percentage}
+            blurb={blurb} />
+        )
+      }
```
Comment (tomer-tgp) [inline app/(company)/about/components/HopeSection.js:66]: {BUILD_WITH_US_CARDS.map(stat => (<HopeStat {...stat} key={stat.header} /> )}

File: app/(company)/about/components/NotPoliticalSection.js [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+        className="mb-4 flex items-center">
+        <PartyLogo
+          name={name}
+          logoFileName={`/images/parties-logos/${logoFileName}`} />
+        <span
+          className="text-sm md:text-base font-sfpro font-medium">{name}</span>
+      </div>
+    )
+  }
+</div>
```
Comment (tomer-tgp) [inline app/(company)/about/components/NotPoliticalSection.js:48]: separate component

**Review by tomer-tgp — CHANGES_REQUESTED** [review]
There are a few types of comments here:
1. changes needed (priority on hero images, split each component into its own file)
2. linting - it seems your IDE is not respecting prettier and esLint config.
3. style - especially usage of grid. Use the grid for all screen sizes and give col-span-12 for most small screens
4. usage of <MaxWidth> as a container instead of duplicating it. (imagine we want to change the max width on all of our pages, we should do it in one place)

File: app/(company)/about/components/AboutHero.js [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+  xl:px-0
+  xl:py-24
+  xl:mx-auto
+  bg-primary-dark">
+    <div className="max-w-screen-xl mx-auto">
+      <div className="text-white
+        font-medium
+        xl:grid
+        xl:grid-cols-12">
+        <div className="mb-8 md:mb-16 xl:mb-0 xl:col-start-1 xl:col-span-6">
```
[the remark this answers is not in the document]
Comment (RavenHursT) [reply app/(company)/about/components/AboutHero.js:16]: 🤔 

Well.. what I have there works for all screen widths.. 

What you're suggesting here gives me the following for the `md` view? 🤷‍♂️ 

image
image

[the remark this answers is not in the document]
Comment (RavenHursT) [reply app/(company)/about/components/AboutHero.js]: https://github.com/thegoodparty/gp-webapp/pull/156/commits/084c5acdb5adc49985068923556004c3c8aa21d5

File: app/(company)/about/components/AboutHero.js [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+            labelId="subscribe-form"
+            label="Join the movement"
+          />
+        </div>
+        <div className="xl:col-start-8 xl:col-span-5">
+          <Image
+            className="w-full hidden md:block"
+            width={832}
+            height={681}
+            src="/images/landing-pages/about-hero.png" alt="about-hero" />
```
[the remark this answers is not in the document]
Comment (RavenHursT) [reply app/(company)/about/components/AboutHero.js]: https://github.com/thegoodparty/gp-webapp/pull/156/commits/7b4b60da9443466ea9c9105b02bb0be7ec781252

File: app/(company)/about/components/AboutHero.js [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+          <Image
+            className="w-full md:hidden"
+            width={288}
+            height={252}
+            src="/images/landing-pages/about-hero-sm.png" alt="about-hero" />
```
[the remark this answers is not in the document]
Comment (RavenHursT) [reply app/(company)/about/components/AboutHero.js]: https://github.com/thegoodparty/gp-webapp/pull/156/commits/7b4b60da9443466ea9c9105b02bb0be7ec781252

File: app/(company)/about/components/AboutHero.js [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+import EmailFormV2 from '@shared/inputs/EmailFormV2';
+
+const AboutHero = () => <section className="relative
+  p-4
+  md:p-8
+  xl:px-0
+  xl:py-24
+  xl:mx-auto
+  bg-primary-dark">
+    <div className="max-w-screen-xl mx-auto">
```
[the remark this answers is not in the document]
Comment (RavenHursT) [reply app/(company)/about/components/AboutHero.js:11]: > and use

And use... what? 🤔

File: app/(company)/about/components/BuildWithUsGridItems.js [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+    <h3
+      className="text-tertiary-dark text-2xl font-medium leading-8 mb-4">{header}</h3>
+    <p>{blurb}</p>
+    <Link className="flex items-center text-tertiary-dark absolute bottom-0"
+          href={href}>
+      <BsFillArrowUpRightCircleFill
+        className="inline mr-2 h-8 w-8" /><span>{linkText}</span>
+    </Link>
+  </div>
+</section>;
```
[the remark this answers is not in the document]
Comment (RavenHursT) [reply app/(company)/about/components/BuildWithUsGridItems.js]: https://github.com/thegoodparty/gp-webapp/pull/156/commits/8439a12fb0bcdde0898dda7272b0b45992e95ada

File: app/(company)/about/components/BuildWithUsSection.js (hunk 2 above)
Comment (RavenHursT) [reply app/(company)/about/components/BuildWithUsSection.js:32]: `npm run lint` exits successfully? 🤷 
image

Comment (RavenHursT) [reply app/(company)/about/components/BuildWithUsSection.js:32]: Same result w/ `next lint --fix`? 🤷 
image

File: app/(company)/about/components/BuildWithUsSection.js (hunk 1 above)
Comment (RavenHursT) [reply app/(company)/about/components/BuildWithUsSection.js:11]: `npm run lint` exits successfully? 🤷 
image

Comment (RavenHursT) [reply app/(company)/about/components/BuildWithUsSection.js:11]: Same result w/ `next lint --fix`? 🤷 
image

File: app/(company)/about/components/HopeSection.js (hunk 3 above)
Comment (RavenHursT) [reply app/(company)/about/components/HopeSection.js]: https://github.com/thegoodparty/gp-webapp/pull/156/commits/704759293e6224323837758f2cc27e8046baae1b

File: app/(company)/about/components/BuildWithUsGridItems.js [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+    src="/images/landing-pages/build-with-us-1.png"
+    alt="Build With Us Person 1" />
+  {
+    BUILD_WITH_US_CARDS.map(
+      ({
+        header,
+        blurb,
+        href,
+        linkText,
+      }, key) => <BuildWithUsCard
```
[the remark this answers is not in the document]
Comment (RavenHursT) [reply app/(company)/about/components/BuildWithUsGridItems.js]: https://github.com/thegoodparty/gp-webapp/pull/156/commits/d193c8b12d847a3f3fa0f07f7ba8e7eb01a4fee8

File: app/(company)/about/components/BuildWithUsSection.js (hunk 2 above)
Comment (RavenHursT) [reply app/(company)/about/components/BuildWithUsSection.js]: https://github.com/thegoodparty/gp-webapp/pull/156/commits/076eaadd2628020763c09b4ce890d8997c691b6f

File: app/(company)/about/components/HopeSection.js (hunk 4 above)
Comment (RavenHursT) [reply app/(company)/about/components/HopeSection.js]: https://github.com/thegoodparty/gp-webapp/pull/156/commits/de0c233b56f775531495827a7e73e838394cfdb1

File: app/(company)/about/components/NotPoliticalSection.js (hunk 5 above)
Comment (RavenHursT) [reply app/(company)/about/components/NotPoliticalSection.js]: https://github.com/thegoodparty/gp-webapp/pull/156/commits/8a51cb42336cbb2b97f6ec4b4881a653ca44b613

File: app/(company)/about/components/AboutHero.js (hunk 9 above)
[the remark this answers is not in the document]
Comment (RavenHursT) [reply app/(company)/about/components/AboutHero.js]: https://github.com/thegoodparty/gp-webapp/pull/156/commits/96947f8c1c79a3c4b8a2c9759b1d18d670d3ca10

File: app/(company)/about/components/PeopleShouldSection.js [hunk 12]
```diff
@@ -0,0 +1,49 @@
+const PeopleShouldSection = () => <section className="relative
+  px-4
+  pt-8
+  md:pt-24
+  md:px-24
+  xl:px-0
+  xl:pt-24
+  xl:mx-auto">
+    <div className="max-w-screen-xl mx-auto">
```
Comment (tomer-tgp) [inline app/(company)/about/components/PeopleShouldSection.js:9]: remove the padding from above and use `MaxWidth` component

File: app/(company)/about/components/NotPoliticalSection.js [hunk 13]
```diff
@@ -0,0 +1,40 @@
...
+
+const NotPoliticalSection = () => <section className="relative
+  px-4
+  mb-12
+  md:px-24
+  md:mb-24
+  xl:px-0
+  xl:mx-auto">
+  <div
+    className="max-w-screen-xl mx-auto bg-secondary-light p-6 md:p-16 rounded-3xl xl:grid grid-cols-12 gap-16">
```
Comment (tomer-tgp) [inline app/(company)/about/components/NotPoliticalSection.js:18]: remove the padding from above and use `MaxWidth` component

**Review by tomer-tgp — APPROVED** [review]

**Pull request merged**